GOVERNANCE.md

```diff
@@ -0,0 +1,143 @@
+# Veraison Governance
+
+The following document outlines Veraison project governance.
+
+## The Veraison Project
+
+The Veraison project consists of several repositories known as sub-projects that enable community cohorts to experiment and implement solutions across the scope of the Veraison project.
+
+## Maintainers Structure
+
+There are two types of maintainers in the Veraison project organized hierarchically.
+Veraison org maintainers oversee the overall project and its health.
+Sub-project maintainers focus on a single codebase or a group of related codebases.
+
+Changes in maintainership must be announced via an [issue][issue].
+
+### Maintainer Responsibility
+
+Veraison maintainers adhere to the requirements and responsibilities set forth in the respective [Veraison Org Maintainers](#veraison-org-maintainers) and [Sub-project Maintainers](#sub-project-maintainers).
+They further pledge the following:
+
+* To act in the best interest of the project and sub-projects at all times.
+* To ensure that project and sub-project development and direction is a function of community needs.
+* To never take any action while hesitant that it is the right action to take.
+* To fulfill the responsibilities outlined in this document and its dependents.
+
+### Veraison Org Maintainers
+
+The [Veraison Org maintainers](./MAINTAINERS.toml) are responsible for:
+
+* Maintaining the mission, vision, values, and scope of the project
+* Refining the governance and charter as needed
+* Making project level decisions
+* Resolving escalated project decisions when the sub-project maintainers responsible are blocked
+* Managing the Veraison brand
+* Controlling access to Veraison assets such as source repositories, hosting, project calendars
+* Deciding what sub-projects are part of the Veraison project
+* Deciding on the creation of new sub-projects
+* Overseeing the resolution and disclosure of security issues
+* Managing financial decisions related to the project
+
+Changes to org maintainers use the following:
+
+* Any sub-project maintainer is eligible for a position as an org maintainer
+* No one company or organization can employ a [simple majority][simple-majority] of the org maintainers
```

Comment:
Noting that this would be a problem (at least for now), given that the four listed maintainers are all Arm employees.

Reply:
see #5 as a start.